Makefile

```diff
@@ -6,7 +6,7 @@ SHELL 			  		= bash -e -o pipefail
 KIND_CLUSTER_NAME 		?= kind
 DOCKER_USER       		?=
 DOCKER_PASSWORD   		?=
-MODEL_COMPILER_VERSION  ?= v0.9.25
+MODEL_COMPILER_VERSION  ?= v0.9.26
```

Comment:
shouldn't we use `0.9.28` which is the one you are releasing in this other PR? https://github.com/onosproject/config-models/pull/238

Reply:
thanks - i'll update it once it's published